internal/clients/provider_config.go

```diff
@@ -51,6 +51,7 @@ const (
 	errAWSConfigUpbound     = "failed to get AWS config using Upbound identity"
 
 	upboundProviderIdentityTokenFile = "/var/run/secrets/upbound.io/provider/token"
+	defaultIdentityTokenFile         = "/var/run/secrets/eks.amazonaws.com/serviceaccount/token"
```

Comment:
It seems that we do not use this constant. Could you please remove it?

Reply:
Done. Thanks for catching it.